backbone/src/main/java/org/researchstack/backbone/model/DocumentProperties.java

```diff
@@ -88,4 +90,24 @@ public void setRequiresBirthdate(boolean requiresBirthdate)
     {
         this.requiresBirthdate = requiresBirthdate;
     }
+
+    public boolean generatePassword()
+    {
+        return generatePassword;
+    }
+
+    public void setGeneratePassword(boolean generate)
```

Comment:
Without any code comments, it is unknown what generate password variable does.  Can you add some comments?